src/BME280Spi.cpp

```diff
@@ -132,3 +132,11 @@ bool BME280Spi::WriteRegister
 
    return true;
 }
+
+
+/****************************************************************/
+bool BME280Spi::IsConnected(void) {
```

Comment:
Since these are defaulted to true, this indicates to me that IsConnected shouldn't be in the base class.

Reply:
That's correct. Now it's implemented much simpler! Thank you for your annotation.